features.json

```diff
@@ -124,6 +124,22 @@
 				"simd": "--enable-simd",
 				"threads": "--enable-threads"
 			}
+		},
+		"Node.js": {
+			"url": "https://nodejs.org/",
+			"logo": "/images/nodejs.svg",
+			"version": "12.18.1",
```

Comment:
I can see benefits of using LTS version for some customers, but TBH I'd just use the "current" stable one (14.4.0) for the feature table instead.

Otherwise, for example, Firefox also has an ESR (Extended Support Release) version that we could use, but I'm not sure it's interesting to *most* consumers.

Reply:
Unlike Firefox though, Node.js does default to the LTS on its download page. I assume that means most users are on that, but I'm not sure how to check?

(I'm also assuming using the version most users are on is best for the table, is that a good assumption?)

Reply:
> (I'm also assuming using the version most users are on is best for the table, is that a good assumption?)

Yeah, that's perfectly reasonable IMO.

> Unlike Firefox though, Node.js does default to the LTS on its download page. I assume that means most users are on that, but I'm not sure how to check?

Hmm, that's a good point. TBH I never visited that page directly, and the home page (which also contains direct download links) shows both as equal: https://nodejs.org/en/

Reply:
> I assume that means most users are on that, but I'm not sure how to check?

Maybe @MylesBorins can help?

Reply:
Meanwhile, do you know what the differences are in terms of feature support between those versions?

Reply:
Meanwhile the best data I can find shows LTS is 55% of users, so it is most popular, but by less than I would have guessed, https://nodejs.org/en/user-survey-report/#version . Latest is 43%.

But the Node.js download page, and top links for "which node.js version?" do point to LTS (e.g. https://tamalweb.com/which-nodejs-version)

Checking the differences as you asked, it turns out there is no difference in features actually (or flag names - only difference I see is the new `--wasm-staging` option was added). Given no differences, showing the older one seems more general?

Reply:
Thanks for the analysis! It seems quite close to 50/50, so even though LTS is slightly more popular according to the survey, I would be still vary of preferring one over another under different circumstances. However...

> Checking the differences as you asked, it turns out there is no difference in features actually (or flag names - only difference I see is the new `--wasm-staging` option was added). Given no differences, showing the older one seems more general?

is exactly the reason I've asked about differences, and I'm glad to hear there isn't any. If so, yeah, I agree we should include LTS / older version here as it's more general.

We'll probably have to revisit this question in the future when stable Node.js gains a new Wasm feature before LTS does, but for now this should be good to go.